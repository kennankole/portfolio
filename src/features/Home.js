const Home = () => (
  <main>
    <h3>Hello and welcome to my page</h3>
  </main>
);

export default Home;
