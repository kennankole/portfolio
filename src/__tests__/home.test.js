/* eslint-disable import/no-extraneous-dependencies */
import Enzyme, { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import Adapter from '@cfaester/enzyme-adapter-react-18';
import Home from '../features/Home';

Enzyme.configure({ adapter: new Adapter() });

it('renders the home page correctly', () => {
  const wrapper = shallow(<Home />);

  expect(toJson(wrapper)).toMatchSnapshot();
});
