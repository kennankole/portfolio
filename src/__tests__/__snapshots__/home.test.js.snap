// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders the home page correctly 1`] = `""`;
